ChangeLog.md

```diff
@@ -113,6 +113,8 @@
   
   * Change element documentation to follow a common structure (James Balamuta).
 
+  * Change JSON validation to use schemas from the [`@prairielearn/schemas`](https://github.com/PrairieLearn/schemas) package (Nathan Walters).
```

Comment:
What's the rational for a new repository, rather than fixing in PrairieLearn or moving to something like PrairieLib? If you're moving the schemas, why not move the code bits too?

Reply:
The schemas and code bits are only really "related" at this point because of how they're used. There are at least three other places I can think of off the top of my head that people might want to consume them from:

* Our VSCode extension
* The online editing experience for in-browser validation
* A custom course CI setup that validates file against our schemas

In those cases, they might want just the schemas without the (minimal) code that does validation and file reading; the first two definitely doesn't need it at all. That's why I opted a) for a second package, and b) to not collocate the validation code with the schemas.

We could definitely move the validation bits to PrairieLib too if you feel they'd be useful in the scheduler or the like. But our functions are already really thin wrappers around standard Node file APIs and a JSON schema validator, of which there are many. I don't know if there's a ton of value in exposing our own functions vs. letting others just pick the tools and read the documentation they want.